enterprise/login-email-password-setup.md

```diff
@@ -0,0 +1,7 @@
+# Login with email and password
+## Private Packagist Enterprise
+
+## Setup
```

Comment:
Maybe add a line that says, by default authentication with email/password is disabled, and tha we recommend setting up one of the OAuth options only if they are available?

Reply:
You mean similar to the note on the installation page? `If your source code is hosted on another platform or you do not with to set up an integration then you can also enable [authentication with email and password](./authentication-email-password-setup.md).`

Reply:
Except nothing about source code? :S Just what I said above?

Reply:
done